docs: updates spelling and grammar (#147)
Manjaro-Sway/manjaro-sway: SUPPORT.md

$unbindsym $mod+w
```

## Upgrades

### how do I upgrade the ~/.config after an update?

After upgrading packages, you will sometimes need to update your skeleton:

```bash
cp -rf /etc/skel/.config/* ~/.config
```

keep in mind that this could cause problems with customizations you made. backup!

### Why are pacman downloads so slow?

You can add fasttrack mirrors using this command:

```bash
sudo pacman-mirrors --geoip && sudo pacman -Syyu
```

> 

Reviewer: ```suggestion
```

> ### How can I update the manjaro-sway-settings package (and all other packages from manjaro-sway)?

Reviewer: ```suggestion
```
not needed anymore (conflicting package removed upstream

> 

Reviewer: ```suggestion
```

> Just prefix the repository using this command: 

Reviewer: ```suggestion
```

> ```bash

Reviewer: ```suggestion
```

> pacman -S manjaro-sway/manjaro-sway-settings

Reviewer: ```suggestion
```

> ```

Reviewer: ```suggestion
```

> ### Why are your packages not in the regular manjaro repos?

Reviewer: ```suggestion
### Why are your packages not in the regular Manjaro Repositories?
```

> We're working on getting them added to the main Manjaro repo. Submissions to them are partly manual and our signing key isn't yet officially part of the manjaro-keyring.

Reviewer: ```suggestion
Submissions to the Manjaro Repositories are partly manual and thus hard to align with our release process. On the contrary, we try to remove our packages from the official package sources again.
```
